apps/dashboard/src/lib/utils/redirect.js

```diff
@@ -0,0 +1,11 @@
+import { goto } from '$app/navigation';
+import { globalStore } from './store/app';
+import { get } from 'svelte/store';
+
+export function redirectToLesson(settings, courseId) {
+  let isOrg = get(globalStore);
```

Comment:
This should not be a `let`

Reply:
and `isOrg ` doesn't match the value returned at all.

It should be something like `globalStoreValue` because `get` returns the value of any given store.